analyzers/src/SonarAnalyzer.Common/Helpers/CountComparisonSyntax.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System;
+using System.Linq;
+
+namespace SonarAnalyzer.Helpers
+{
+    public readonly struct CountComparisonSyntax : IEquatable<CountComparisonSyntax>
+    {
+        public enum Kind
+        {
+            Equals,
+            NotEquals,
+            LessThan,
+            LessThanOrEqual,
+            GreaterThan,
+            GreaterThanOrEqual,
+        }
+
+        public CountComparisonSyntax(int? left, Kind @operator, int? right)
```

Comment:
Please try to sort all of these members according to our https://github.com/SonarSource/sonar-dotnet/blob/master/docs/coding-style.md

and add empty lines between methods. Having block of properties without empty lines between them is fine.